api/src/main/java/io/kaoto/backend/api/resource/v2/IntegrationsResource.java

```diff
@@ -146,7 +146,8 @@ private static void ensureUniqueNames(FlowsWrapper answer) {
             }
             //Make sure there is an id/name assigned to all flows
             if (!flow.getMetadata().containsKey(name)) {
-                flow.getMetadata().put(name, flow.getDsl() + System.currentTimeMillis());
+                flow.getMetadata().put(name,
+                        flow.getDsl().toLowerCase().replaceAll(" ", "") + System.currentTimeMillis());
```

Comment:
Instead of using a timestamp, could we use a 4 digits random number instead?

Reply:
Timestamp is sure to not be repeated, I feel safer with timestamp :laughing:

Reply:
Is it very ugly on the frontend side then?

Reply:
Added a random integer.